### PR TITLE
Writing refinements for the README

### PR DESCRIPTION
**Pull request by danielfdickinson**
Excellent work! I've made a few minor suggestions. Once I've created the PR I'll annotate some spots where what is ideal is a matter of taste and/or you may disagree with my changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,38 +10,40 @@ Filter, sort, preview, and find the best Jamstack Theme for your next project!
...
 ### Report a problem
 
 Found an issue? You can [submit a bug report](https://github.com/RoneoOrg/jamstack.club/issues), or [drop us an email](https://jamstack.club/contact/).
 
 We'll get back to you as soon as possible.
+
 ### Submit a new theme
 
-A theme must meet only two requirements to be submitted:
+There are only two requirements for submitting a theme:
```
Comment (danielfdickinson) [inline README.md:23]: This could perhaps be: The following are the only only two requirements for submitting a theme.

File: README.md [hunk 2]
```diff
@@ -10,38 +10,40 @@ Filter, sort, preview, and find the best Jamstack Theme for your next project!
...
 
 * The source code must be open-source and public
 * The demo URL links must point to a demo of the theme and not a personal/business site.
 
-Allright? Let's go!
+Ready? Let's go!
```
Comment (danielfdickinson) [inline README.md:28]: I'm not sure I like 'Ready?' but 'Allright?' doesn't work me. It should at least be: "All right?" or "Alright?".

File: README.md [hunk 3]
```diff
@@ -10,38 +10,40 @@ Filter, sort, preview, and find the best Jamstack Theme for your next project!
...
 
-1. Open [this template file](https://github.com/RoneoOrg/jamstack.club/edit/main/content/theme/theme-contribution-template.md) and confirm the creation of a Fork.  
+1. Open [this template file](https://github.com/RoneoOrg/jamstack.club/edit/main/content/theme/theme-contribution-template.md) and confirm the creation of a fork.
```
Comment (danielfdickinson) [inline README.md:30]: It's not a proper noun (that is not the _name_ of place, person, or thing) that I know about.

File: README.md [hunk 4]
```diff
@@ -10,38 +10,40 @@ Filter, sort, preview, and find the best Jamstack Theme for your next project!
...
-1. Open [this template file](https://github.com/RoneoOrg/jamstack.club/edit/main/content/theme/theme-contribution-template.md) and confirm the creation of a Fork.  
+1. Open [this template file](https://github.com/RoneoOrg/jamstack.club/edit/main/content/theme/theme-contribution-template.md) and confirm the creation of a fork.  
 (See [the Github documentation](https://docs.github.com/en/repositories/working-with-files/managing-files/editing-files#editing-files-in-another-users-repository) for details, and reach out if you're stuck somewhere!)
 2. Edit the file and click on "Propose changes" at the bottom.
 3. Submit the Pull Request.
-4. You're done!  
-Yes, the Screenshot and the Demo page, the Github Stars.. everything is generated automatically!
+4. You're done!
+
+Yes, the screenshot and the demo page; the Github stars — everything is generated automatically!
```
Comment (danielfdickinson) [inline README.md:36]: I'm not sure how to do this sentence. I actually rather liked what you had, but it wasn't a proper sentence, and needs a little tweaking even if you go that route.

File: README.md [hunk 5]
```diff
@@ -10,38 +10,40 @@ Filter, sort, preview, and find the best Jamstack Theme for your next project!
...
 
 ## How does it work?
 
-Each theme is described in a Markdown file. (See [this example](https://raw.githubusercontent.com/RoneoOrg/jamstack.club/main/content/theme/hugo-restaurant.md) to get an idea).
+Each theme is described in a Markdown file. (See [this example if a theme specification](https://raw.githubusercontent.com/RoneoOrg/jamstack.club/main/content/theme/hugo-restaurant.md) to get an idea).
```
Comment (danielfdickinson) [inline README.md:40]: Should of 'of' not 'if' in the link text (that is, I goofed). In general link text should be relatively descriptive on it's own (according to the various accessibility standards).

File: README.md [hunk 6]
```diff
@@ -10,38 +10,40 @@ Filter, sort, preview, and find the best Jamstack Theme for your next project!
...
+The website is built with [Hugo](https://gohugo.io) along with [a few JS scripts](https://github.com/RoneoOrg/jamstack.club/tree/main/scripts) to automate the creation of the Github stats and generate a screenshot.
 
-A [dedicated page](https://jamstack.club/theme/hugo-restaurant/) is created for each theme, along with [a Demo page](https://jamstack.club/demo/theme/hugo-restaurant/), embedding the Demo URL filled out in the Markdown file.
+A [dedicated page](https://jamstack.club/theme/hugo-restaurant/) is created for each theme, as well as [a theme demo website](https://jamstack.club/demo/theme/hugo-restaurant/), at the 'Demo' URL filled out in the Markdown file.
 
-The index is updated, still automatically, and the website is published, hosted by [Netlify](https://www.netlify.com/), landing at [Jamstack.club](https://jamstack.club).
+The index is updated, still automatically, and the website is published. It is hosted by [Netlify](https://www.netlify.com/) with a landing at [Jamstack.club](https://jamstack.club).
```
Comment (danielfdickinson) [inline README.md:46]: 'with a landing at' should maybe be: 'and now has a landing page at'.

**Review by danielfdickinson — COMMENTED** [review]
I've been verbose with my explanations and where I'm not sure of the best way of saying what is being said. Hope it helps!

File: README.md (hunk 1 above)
Comment (RoneoOrg) [reply README.md:23]: I'm totally fine with your first suggestion

File: README.md (hunk 2 above)
Comment (RoneoOrg) [reply README.md:28]: Sure, "Ready?" is definitively more logical

File: README.md (hunk 3 above)
Comment (RoneoOrg) [reply README.md:30]: 👍

File: README.md (hunk 4 above)
Comment (RoneoOrg) [reply README.md:36]: I like it this way! Is it correct to use a semicolon here, or is it a typo?

File: README.md (hunk 5 above)
Comment (RoneoOrg) [reply README.md:40]: You're absolutely right, thanks!
(The typo was fixed in https://github.com/RoneoOrg/jamstack.club/commit/e44779c56c947a7557233a5680af92efc93b37d6)

File: README.md (hunk 6 above)
Comment (RoneoOrg) [reply README.md:46]: I like the short version, it looks like it was written with someone familiar with english ;)

**Comment by RoneoOrg** [comment]
This PR was merged upstream: https://github.com/RoneoOrg/jamstack.club/pull/23
Thanks, Daniel!

File: README.md (hunk 4 above)
Comment (danielfdickinson) [reply README.md:36]: I'm not 100% sure, but can't think of a way to write it that remains 'fun' and I am certain is pedantically correct, so I vote for keeping it this way 🙂

Comment (RoneoOrg) [reply README.md:36]: :D Let's go for it, then!

**Comment by danielfdickinson** [comment]
Merged upstream, so closing here and pulling from upstream instead of merging here.

**Pull request closed without merging**